strr-host-pm-web/app/stores/propertyRequirements.ts

```diff
@@ -0,0 +1,146 @@
+import { z } from 'zod'
+import { FetchError } from 'ofetch'
+
+interface PropertyRequirements {
+  isBusinessLicenceRequired: boolean
+  isPrincipalResidenceRequired: boolean
+  isStrProhibited: boolean
+  isStraaExempt: boolean | null
+  organizationNm: string
+}
+
+interface PropertyRequirementsError {
+  error?: FetchError
+  type: 'fetch' | 'unknown' // TODO: handle other error types
+}
+
+interface PrRequirements {
```

Comment:
Probably want to export these so that we can use them for when we build the display components for the data later. Are you planning to move these to the 'interfaces' folder or do you like having them within the store? I think there's a good argument for either place

Reply:
I'm planning to move them before finalizing the pr